docs/source/examples/aequilibrae_project/plot_scenarios.py

```diff
@@ -0,0 +1,167 @@
+"""
+# %%
+# .. admonition:: References
+#
+#   * :doc:`../../aequilibrae_project`
+
+# %%
+# .. seealso::
+#     Several functions, methods, classes and modules are used in this example:
+#
+#     * :func:`aequilibrae.Project.list_scenarios`
+#     * :func:`aequilibrae.Project.switch_scenario`
+#     * :func:`aequilibrae.Project.create_empty_scenario`
+#     * :func:`aequilibrae.Project.clone_scenario`
+
+# %%
+
+# Imports
+from uuid import uuid4
+from tempfile import gettempdir
+from pathlib import Path
+from os.path import join
+import pandas as pd
+
+from aequilibrae.utils.create_example import create_example
+from aequilibrae import TrafficAssignment, TrafficClass
+
+# sphinx_gallery_thumbnail_path = '../source/_images/plot_scenarios.png'
+
+# %%
+
+# We create the example project inside our temp folder.
+fldr = Path(gettempdir()) / uuid4().hex
+project = create_example(fldr, "sioux_falls")
+
+
+# %%
+# Working with scenarios
+# ----------------------
+# Let's first see what scenarios exist in our project
+
+project.list_scenarios()
+
+# %%
+# The root scenario is always present and represents the base model.
+# Let's examine the current scenario's network
+
+print(f"Current scenario network has {len(project.network.links.data)} links")
+print(f"Current scenario network has {len(project.network.nodes.data)} nodes")
+
+# %%
+# Creating new scenarios
+# ----------------------
+# We can create empty scenarios or clone existing ones
+
+# Create an empty scenario for testing modifications
+project.create_empty_scenario("test_modifications", "Scenario for testing network modifications")
+
+# Clone the root scenario to preserve the original network
+project.clone_scenario("alternative_parameters", "Testing different assignment parameters")
+
+# %%
+# Let's see our updated scenario list
+
+project.list_scenarios()
+
+# %%
+# Switching between scenarios
+# ---------------------------
+# Each scenario operates independently with its own data
+
+# Switch to the cloned scenario
+project.switch_scenario("alternative_parameters")
+print(f"This scenario has {len(project.network.links.data)} links")
+
+# %%
+# Let's perform a traffic assignment in this scenario with different parameters
+
+# Build the network graph
+project.network.build_graphs(fields=["distance", "capacity_ab", "capacity_ba"], modes=["c"])
+graph = project.network.graphs["c"]
+graph.set_graph("distance")
+graph.set_blocked_centroid_flows(False)
+
+# Get the demand matrix
+mat = project.matrices.get_matrix("demand_omx")
+mat.computational_view()
+
+# Create traffic assignment with alternative parameters
+assigclass = TrafficClass("car", graph, mat)
+assignment = TrafficAssignment(project)
+assignment.add_class(assigclass)
+assignment.set_vdf("BPR")
+
+# Use different VDF parameters than we might use in the root scenario
+assignment.set_vdf_parameters({"alpha": 0.25, "beta": 6.0})  # More congested parameters
```

Comment:
This is not something people do in practice, hence the suggestion above to change the network itself.